test/src/test/scala/test/raphtory/DynamicClassLoaderTest.scala

```diff
@@ -145,8 +150,6 @@ class DynamicClassLoaderTest extends CatsEffectSuite {
 
   case object RemoteContext extends Context
 
-  val defaultConf: Config = ConfigBuilder.getDefaultConfig
```

Comment:
Do you mind updating the context creation below to use the new `RaphtoryIOContext.localIO()` and `RaphtoryIOContext.remoteIO()` methods?